sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -108,9 +108,16 @@ case class CreateViewCommand(
     verifyTemporaryObjectsNotExists(catalog)
 
     if (viewType == LocalTempView) {
+      if (replace && catalog.getTempView(name.table).isDefined) {
+        sparkSession.catalog.uncacheTable(name.quotedString)
```

Comment:
How about showing an INFO log for this case instead of implicitly uncaching?

Reply:
I prefer to adding INFO log, but not removing the `uncacheTable` here, because after replacing, there is no way to drop the old cached data.

Reply:
I didn't mean so and its ok to just add logging.

Reply:
Yes. Added.